app/controllers/experiments_controller.rb

```diff
@@ -243,6 +243,8 @@ def create_experiment
           format.json { render json: {status: 'error', message: flash[:error]} }
         end
       else
+        # TODO: provide proper algorithm once we have any, get interval and algorithm params from form
+        WorkersScaling::AlgorithmRunner.new(experiment.id, nil, 10).start
```

Comment:
Przydałby się jakiś if czy jest obecny parametr, w tym momencie jeżeli to się wywali nie uruchomisz żadnego eksperymentu.